Google.Api.Generator/Generation/MethodDetails.cs

```diff
@@ -394,6 +395,9 @@ IEnumerable<string> ExtractBracedPaths(string s)
         /// <summary>The service in which this method is defined.</summary>
         public ServiceDetails Svc { get; }
 
+        /// <summary>The name of this method in the method descriptor</summary>
+        public string MethodName { get; }
```

Comment:
nit: maybe ProtoMethodName, or anything else that makes it inmediately clear that is a name from the input and not a name to be used in the output.

Reply:
Or possibly `RpcName`? Given that they're declared as `rpc` in the proto file? Don't mind either way - but I agree that it's good to clarify that it's from the proto.

Reply:
`ProtoRpcName` it is then

Reply:
👍